### PR TITLE
Copy Config from Appx to LocalStateFolder

### PR DESCRIPTION
**Pull request by splumber**
## Purpose
* config.json is copied and read from AppData LocalState Folder
* kws models in keyword-models folder are copied to the SDKKeywords folder which is then copied and read from AppData LocalState Folder

## Does this introduce a breaking change?
[x] Yes
[ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:

## How to test

* I have included a computer.txt file with the necessary credentials for computer and contoso. 
* Replace contoso with computer in config.json and relaunch the app. The new keyword will be registered

### CODE REVIEW

File: clients/csharp-uwp/UWPVoiceAssistantSample/AppSettings.cs [hunk 1]
```diff
@@ -62,12 +84,13 @@ public class AppSettings
...
+        public static async Task<AppSettings> Load(StorageFile configFile)
         {
-            StreamReader file = new StreamReader(configFile);
-            string config = file.ReadToEnd();
-            file.Close();
-            AppSettings instance = JsonConvert.DeserializeObject<AppSettings>(config);
+            //StreamReader file = new StreamReader(configFile);
+            string file = await Windows.Storage.FileIO.ReadTextAsync(configFile);
+            //string config = file.ReadToEnd();
+            //file.Close();
```
Comment (cfogg6) [inline clients/csharp-uwp/UWPVoiceAssistantSample/AppSettings.cs:92]: I'm not super familiar with c#/uwp file io- do we not need the closefile anymore?

Comment (splumber) [reply clients/csharp-uwp/UWPVoiceAssistantSample/AppSettings.cs:92]: I used this page as guidance. Docs does not indicate closing of file and no close method exists on the file.

https://docs.microsoft.com/en-us/windows/uwp/files/quickstart-reading-and-writing-files

Comment (dargilco) [reply clients/csharp-uwp/UWPVoiceAssistantSample/AppSettings.cs:92]: Are the commented lines going to be removed?

File: clients/csharp-uwp/UWPVoiceAssistantSample/KeywordRegistration.cs [hunk 2]
```diff
@@ -331,9 +349,12 @@ private async Task SetModelDataIfNeededAsync(ActivationSignalDetectionConfigurat
             }
         }
 
+        //private async Task<StorageFile> GetFileFromPathAsync(string path)
+        //    => path.StartsWith("ms-appx", StringComparison.InvariantCultureIgnoreCase)
+        //        ? await StorageFile.GetFileFromApplicationUriAsync(new Uri(path))
+        //        : await StorageFile.GetFileFromPathAsync(path);
+
```
Comment (dargilco) [inline clients/csharp-uwp/UWPVoiceAssistantSample/KeywordRegistration.cs:356]: Will commented lines be removed?

**Pull request merged**